src/config.js

```diff
@@ -35,6 +35,11 @@ const DefaultConfig = {
     persistence: false,
     threshold: 5
   },
+  peerRouting: {
+    enabled: true,
+    interval: 300e3,
+    bootDelay: 10e3
+  },
```

Comment:
This is a bit confusing as it's not quite peer routing. This is really the refresh manager. In go-libp2p since they just have the DHT right now they run the table refreshes when they search for their closest peers. This happens when DHT bootstrap is run at startup and then on an interval. Ideally we would not use a boot delay timer as this is finicky, and instead run once we've finished a bootstrap phase. Since we don't really have this concept atm, I think a delay is fine for now.

Go runs this every 10 minutes by default right now, we should just match that for consistency.

Reply:
I am going to change this into 10 minutes yes. Regarding the namings, thinking on changing this to:
```
peerRouting: {
  refreshManager: { ... }
}
```

The peerRouting is the scope where we make these queries, so it seems that this configs should live bound to it. What do you think?

Reply:
Added as suggested for now